tests/keras_autolog/test_keras_autolog.py

```diff
@@ -186,18 +186,19 @@ def keras_random_data_run_with_callback(
     model = create_model()
     if callback == "early":
         # min_delta is set as such to guarantee early stopping
-        callback = keras.callbacks.callbacks.EarlyStopping(
+        callback = keras.callbacks.EarlyStopping(
```

Comment:
`keras.callbacks.EarlyStopping` is valid for the versions of Keras we support, whereas `keras.callbacks.callbacks.EarlyStopping` was removed / renamed in 2.4.0.